assets/admission-webhook/deployment.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
     app.kubernetes.io/managed-by: cluster-monitoring-operator
     app.kubernetes.io/name: prometheus-operator-admission-webhook
     app.kubernetes.io/part-of: openshift-monitoring
-    app.kubernetes.io/version: 0.63.0
+    app.kubernetes.io/version: 0.65.1
```

Comment:
@marioferh ,  suggestion for next time. Could you please keep the unrelated changes in a separate commit?

Reply:
sure, sorry it was make versions but now I know it is not necessary :)

Reply:
@marioferh can you revert the changes to jsonnet/versions.yaml and regenerate the assets? it would indeed help reviewing the PR.